.travis.yml

```diff
@@ -3,38 +3,45 @@ language: python
 matrix:
   include:
     - python: 2.6
-      sudo: false
     - python: 2.7
-      sudo: false
     - python: 3.3
-      sudo: false
     - python: 3.4
-      sudo: false
     - python: 3.5
-      sudo: false
+    - python: 3.6
+    - python: 3.7
     - python: pypy
-      sudo: required
-      dist: trusty
     - python: pypy3
-      sudo: required
-      dist: trusty
+
+cache:
+  directories:
+    - $HOME/.cache/pip
+
+branches:
```

Comment:
I think we probably want to run all branches. Is there a reason to limit to master only?

Reply:
This option will limit builds to PRs and `master` brach updates. Otherwise you get two separate builds when updating a PR branch.